mne/preprocessing/ica.py

```diff
@@ -133,11 +133,16 @@ class ICA(ContainsMixin):
         >> ica.fit(raw)
         >> raw.info['projs'] = projs
 
-    .. note:: Methods implemented are FastICA (default), Infomax and
-              Extended-Infomax. Infomax can be quite sensitive to differences
-              in floating point arithmetic due to exponential non-linearity.
-              Extended-Infomax seems to be more stable in this respect
-              enhancing reproducibility and stability of results.
+    .. note:: Methods implemented are FastICA (default), Infomax,
+              Extended-Infomax, and Picard. Infomax can be quite sensitive to
+              differences in floating point arithmetic due to exponential
```

Comment:
Are we 100% sure it is due to the exponential non-linearity? Maybe just remove "due to exponential non-linearity".

Reply:
Yes I don't think that's the reason either, it's just been in the note since the beginning.

Reply:
I know, now is a good time to get rid of it :smile: